packages/application-components/src/components/detail-pages/custom-form-detail-page/custom-form-detail-page.tsx

```diff
@@ -87,18 +88,22 @@ const CustomFormDetailPage = (props: CustomFormDetailPageProps) => {
             previousPathLabel={props.previousPathLabel}
             onClick={props.onPreviousPathClick}
           />
-          {props.customTitleRow || (
-            <PageHeaderTitle
-              title={props.title ?? ''}
-              subtitle={props.subtitle}
-              titleSize="big"
-            />
-          )}
-          {!props.hideControls && props.formControls && (
-            <Spacings.Inline justifyContent="flex-end">
-              {props.formControls}
-            </Spacings.Inline>
-          )}
+          <Spacings.Inline scale="l" justifyContent="space-between">
```

Comment:
This is not correct. The `customTitleRow` should be in its own row.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/1110551/191449877-d14dbef3-27ed-4bc0-a9b3-b116516b5fd6.png">